packages/core/src/codewhisperer/service/transformationHubViewProvider.ts

```diff
@@ -248,4 +258,14 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
             return `<span style="color: grey"> ✓ </span>`
         }
     }
+
+    private getFontColorForSubStep(status: string) {
```

Comment:
Question: For IntelliJ we use native colors. Those native colors depend on the dark/light theme displaying. 

Do we need a tertiary statement here like the following pseudo code:
```
return isLightTheme() ? VSCode.colors.light.green : VSCode.colors.dark.green
```

Reply:
![image](https://github.com/aws/aws-toolkit-vscode/assets/60020664/3d6b4235-7a13-4d22-a0fb-4a0bb7b1ad9a)
![image](https://github.com/aws/aws-toolkit-vscode/assets/60020664/e6372982-fa52-4d22-bb85-d78e2447347d)

Reply:
Looks like there are dozens of different theme options, each of which may use their own shade of red/green, so using generic red/green as I've done here is probably best. Don't think it's worth extracting the theme the user is using, determining if light/dark/other, then making a color decision, that's kind of overkill IMO.

Reply:
Not a blocker, just a comment. I don't think its overkill. We should definitely have a mechanism in the future for getting a certain color palette based on light/dark themes.

For IntelliJ we just covered:

    light themes get a darker red/green
    dark themes get a light red/green

We did not cover individual themes, because there are too many and custom ones. We went with a "good enough" approach covering the base dark/light themes. I think for VSCode we can do the same.